### PR TITLE
fixes #6363 - make hostgroup provisioning work

### PR DESCRIPTION
**Pull request by stbenjam**
Requires  https://github.com/theforeman/foreman/pull/1634 in order to have the latest version of SafeMode with fixed respond_to?

Note: This breaks Foreman < 1.5.3, but needs to be done to fix host group provisioning.

### CODE REVIEW

**Comment by mmoll** [comment]
what's the status here?

**Comment by stbenjam** [comment]
No idea, it was ready and never got merged.

**Comment by stbenjam** [comment]
To be honest, I've given up on fixing hostgroup provisioning.  It's just not considered at all when any changes are made to Foreman, and it's easy to break because of that.  https://github.com/theforeman/foreman/pull/2114 has also sat for quite some time, with no feedback.

Sooooo, yea, unless it becomes a priority for Foreman core devs to care about it, I'm not going to keep spinning my wheels.

**Comment by mmoll** [comment]
@stbenjam thanks, closing for now.

File: kickstart/provision.erb [hunk 1]
```diff
@@ -151,10 +152,11 @@ salt-call --no-color --grains >/dev/null
 
 sync
 
+<% if host_provision -%>
 # Inform the build system that we are done.
 echo "Informing Foreman that we are built"
 wget -q -O /dev/null --no-check-certificate <%= foreman_url %>
-# Sleeping an hour for debug
```
Comment (ekohl) [inline kickstart/provision.erb]: Removing this comment sounds like a good change in itself since it's a leftover debug comment.

Comment (ekohl) [inline kickstart/provision.erb]: https://github.com/theforeman/community-templates/pull/161

**Comment by lzap** [comment]
@stbenjam sorry it took so long for us to take a look. This should really not happen. I am going to review and test all the realted patches, if you are willing to get back to this. These kind of bugs are real pain both upstream and downstream, got two customer issues now.

**Comment by stbenjam** [comment]
@lzap Not really interested in maintaining this, for example, things I already fixed are just broke again - http://projects.theforeman.org/issues/9773.  It's too time consuming.

**Pull request closed without merging**